Minor fixes (#10)
eficode-academy/rf-advanced-web-katas: exercises/00-getting-started.md

A general overview of the exercise is given first, followed by more detailed step-by-step instructions.
If you want a little challenge, you can try to just read the general steps of the exercise and if you
need more help, take a look at the step-by-step instructions. The step-by-step instructions also
include explanations in some places about _why_ things are done as they are done in the exercise.

The step-by-step instructions are divided into SeleniumLibrary and Browser sections separately.

**Each exercise is summarized in bold text at the beginning.**

All exercises have some explanation on why some things are done as
they are in normal paragraphs.

> - All exercise steps are in bullet points. Each bullet points is an individual small task summary.

Reviewer: ```suggestion
- All exercise steps are in bullet points. Each bullet point is an individual small task summary.
```

Author: We should rephrase this, I am aware of the >>> syntax, and it is mentioned in the later exercises. The mechanism is not the same though, as in Browser you need to add the id=iframe >>> part before every locator. Thus the point of exercise 04 (create keyword that takes another keyword as argument) is lost.

Between the bullet points there are longer descriptive paragraphs.
 **To complete the exercise, it's ok to skip those paragraphs.**

> Read the paragraphs for more explanations and the solution narrative, so you can understand and apply it in your work.

Reviewer: I do not understand what "solution narrative" means in this sentence. Maybe just:
```suggestion
Read the paragraphs for more explanation, so you can understand and apply it in your work.
```
?

Author: Since there is no katas here to check results, in order to help people progress but to avoid giving answers directly, there is a narrative that supports every exercise guiding participants through logic of how the exercises could be solved.

- Run `which python3` to show the location of you Python installation.
- Run `cp /path/to/your/chromedriver /path/to/your/bin`.

Alternatively, on Unix, you can create a new directory, copy webdrivers into it and run
 `export PATH=$PATH:/directory-path`. You will either need to run this in every terminal you use,

>  or add it to your terminal config files.

Reviewer: I feel this would be more accurate:
```suggestion
 or add it to your terminal configuration.
```

Author: Ok, we can use this.
